SplunkRumWorkspace/SplunkRum/SplunkRum/AppLifecycle.swift

```diff
@@ -19,6 +19,19 @@ import Foundation
 import UIKit
 import OpenTelemetrySdk
 
+let INACTIVITY_SESSION_TIMEOUT_SECONDS = 15 * 60
+var IS_NEW_SESSION_ID_FOR_INACTIVITY = false
+private var sessionIdInActivityExpiration = Date().addingTimeInterval(TimeInterval(INACTIVITY_SESSION_TIMEOUT_SECONDS))
```

Comment:
Why the odd capitalization of `InActivity` here?

Reply:
No reason.  I think that is what the variable was named in the other MR.  I will fix the capitalization.